admin/sql/create_tables.sql

```diff
@@ -0,0 +1,33 @@
+BEGIN;
+
+CREATE TABLE lowlevel (
+  id          SERIAL,
+  mbid        UUID NOT NULL,
+  build_sha1  TEXT NOT NULL,
+  lossless    BOOLEAN                  DEFAULT 'n',
+  data        JSON NOT NULL,
+  submitted   TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
+  data_sha256 TEXT NOT NULL
+);
+
+CREATE TABLE highlevel (
```

Comment:
I'm confused - can someone explain to me how create_tables should work in conjunction with the other update sql scripts? Should create_tables always make the most up-to-date version, and people with older versions run the update script to get up-to-date?